prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.html

```diff
@@ -79,6 +79,20 @@
       </td>
     </ng-container>
 
+    <ng-container matColumnDef="allowRequestRemoteAccess">
+      <th mat-header-cell
+          *matHeaderCellDef
+          scope="col"
+          class="prefix-header pl-3">
+        ARRA
+      </th>
+      <td mat-cell
+          class="pl-3"
+          *matCellDef="let row;">
+        {{ row.allowRequestRemoteAccess }}
+      </td>
+    </ng-container>
+
```

Comment:
Unless there is a good reason, this new column should go at the end of the table since the viewers of this table expect columns in a certain order

Reply:
the column is moved to the end of the table.